CHANGELOG.md

```diff
@@ -8,33 +8,55 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
+- `Internal.Tokenizer.Exception`: minified scripts or files should have original files included. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#stylesheets-and-scripts).
+- `Generic.Files.ByteOrderMark`: no ByteOrderMark allowed - important to prevent issues with content being sent before headers
+- `Generic.CodeAnalysis.EmptyStatement`: PHP tags without anything between them is just sloppy
+- `WordPress.WP.I18n`: check that the I18N functions are used correctly. This sniff can also check the text domain, provided it's passed to `PHPCS`.
+- `WordPress.WP.EnqueuedResources`: hard coding of scripts and styles is prohibited. They should be enqueued.
+- `WordPress.Security.PluginMenuSlug`: prevent path disclosure when using add_theme_page().
+- `Generic.PHP.NoSilencedErrors`: usage of Error Control Operator `@` is forbidden in a theme.
+- `WordPress.DB.RestrictedClasses`: the WP abstraction layer should be used to query database if needed.
+- `WordPress.DB.RestrictedFunctions`: the WP abstraction layer should be used to query database if needed.
+- `WordPress.DB.PreparedSQL`: all SQL queries should be prepared as close to the time of querying the database as possible.
+- `WordPress.DB.PreparedSQLPlaceholders`: verify that placeholders in prepared queries are used correctly.
+- `WordPress.Security.ValidatedSanitizedInput`: validate and/or sanitize untrusted data before entering into the database.
+- `WordPress.Security.EscapeOutput`: all untrusted data should be escaped before output - warning, since translations don't have to be escaped.
+- `Generic.PHP.BacktickOperator`: prohibit the use of the backtick operator.
+- `WordPress.WP.GlobalVariablesOverride`: šrohibit overwriting of WordPress global variables.
+- `Squiz.PHP.Eval.Discouraged`: prohibit the use of the eval() PHP language construct.
+- `Generic.PHP.DiscourageGoto.Found`: prohibit the use of the `goto` PHP language construct.
+- `WordPress.WP.DeprecatedClasses`: check for use of deprecated WordPress classes.
+- `WordPress.WP.DeprecatedFunctions`: check for use of deprecated WordPress functions.
+- `WordPress.WP.DeprecatedParameters`: check for use of deprecated WordPress function parameters.
+- `WordPress.WP.DiscouragedConstants`: check for deprecated WordPress constants.
+- `WordPress.NamingConventions.PrefixAllGlobals`: verify that everything in the global namespace is prefixed. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#code).
+- `WordPress.WP.CapitalPDangit`: check for correct spelling of WordPress. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#naming)
+- `WordPress.WP.TimezoneChange`: themes should never touch the timezone.
 
 [Unreleased]: https://github.com/WPTRT/WPThemeReview/compare/master...HEAD
 [0.1.0]: https://github.com/WPTRT/WPThemeReview/compare/tag...0.1.0
```

Comment:
This link should be removed as it won't work, was just there as an example for future releases.